deep-learning/_posts/2017-05-17-tiramisu.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: review
+title:  "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+tags:   deep-learning CNN segmentation
+author: Clément Zotti
+pdf:   "https://arxiv.org/pdf/1611.09326.pdf"
+
+cite:
+  authors: "S.Jégou, M.Drozdzal, D.Vasquez, A.Romero, Y.Bengio"
+  title:   "The One Hundred Layers Tiramisu: Fully Convolutional DenseNets for Semantic Segmentation"
+  venue:   "arXiv 2016"
+---
+
+This papers implement all the well known advanced in the image segmentation community and make it work together.
+
+List of the methods used:
+
+* Dense blocks
+* U-Net architecture
+* Using IOU loss
+
+
+## Dense blocks
+
+The dense block provide a low increase of parameters when keeping a good feature extraction.
```

Comment:
It mean that you have less feature maps than other big network and still extract meaningfull features by leveraging the skip connections in the dense block.